migtests/scripts/live-migration-fallf-run-test.sh

```diff
@@ -89,6 +89,11 @@ main() {
 	fi
 	./init-db
 
+	if [ "${SOURCE_DB_TYPE}" = "postgresql" ]; then
+		step "Creating pg_stat_statements for the compare-performance command"
+		run_psql ${SOURCE_DB_NAME} "CREATE EXTENSION IF NOT EXISTS pg_stat_statements;"
+	fi
```

Comment:
Hmm it's safe to create pg_stat_statements extension for the PostgreSQL source always, irrespective of test. 

Let's place it in `create_source_db` in `function.sh` beside the creation of the database itself.